src/inet/tests/TestInetEndPoint.cpp

```diff
@@ -132,12 +132,9 @@ static void TestResolveHostAddress(nlTestSuite * inSuite, void * inContext)
     char testHostName2[20] = "127.0.0.1";
     char testHostName3[20] = "";
     char testHostName4[260];
-    struct timeval sleepTime;
     IPAddress testDestAddr[1] = { IPAddress::Any };
     CHIP_ERROR err;
-
-    sleepTime.tv_sec  = 0;
-    sleepTime.tv_usec = 10000;
+    constexpr uint32_t kSleepTimeMilliseconds = 100;
```

Comment:
```suggestion
    constexpr uint32_t kSleepTimeMilliseconds = 10;
```
To match the old behavior, right?

Reply:
Indeed, I carelessly bulk-copied from TestInetCommonPosix.cpp where it's 100.